src/main/kotlin/com/goldenraven/tatooine/TatooineWallet.kt

```diff
@@ -74,18 +72,16 @@ object TatooineWallet {
     }
 
     public fun sendTo(address: String): String {
-        val addresseesAndAmounts: List<Pair<String, String>> = listOf(Pair(address, 21000.toString()))
-        var transactionDetails=createTransaction(1F,addresseesAndAmounts, false, null, null, null)
+        val addresseesAndAmounts: List<Pair<String, String>> = listOf(Pair(address, amountToSend))
+        var transactionDetails=createTransaction(feeRate,addresseesAndAmounts, false, null, null, null)
```

Comment:
Missing space between arguments, and between variable name declaration and assignment.